### ISSUE TITLE
Autocomplete attribute to allow for synonyms of on/off

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3188]
This should't be failed:

```html
<input autocomplete="none" />
```

Unfortunately, the only way I can think that we allow this is to create a list of synonyms that someone might mistakenly be using, instead of using "on" or "off" which are actually allowed. So things like "none", "false", "true", "disabled", "enabled", "undefined", "null", etc. They'll need to be in a check option, so that this can be configured.

I don't think we should even bother to report them as incomplete. Passing the rule is fine in this scenario.

**Comment by ljharb** [comment]
For posterity: this broke eslint-plugin-jsx-a11y's tests; specifically, we were asserting that `<input type={isEmail ? "email" : "text"} autocomplete="none" />` is invalid, and v4.4 made it valid.

I don't think this is a breaking change, exactly, because it turned an error into a non-error, but it's worth keeping in mind for the future.
